crates/holochain_types/src/app.rs

```diff
@@ -475,25 +508,68 @@ impl InstalledAppCommon {
         &self.role_assignments
     }
 
-    /// Add a cloned cell
-    pub fn add_clone(&mut self, role_id: &AppRoleId, cell_id: CellId) -> AppResult<()> {
-        let role = self.role_mut(role_id)?;
+    /// Get the next clone index for a role id.
+    pub fn next_clone_index(&self, role_id: &AppRoleId) -> AppResult<u32> {
+        Ok(self.role(role_id)?.next_clone_index)
+    }
+
+    /// Increment index for next clone cell for a role id.
+    pub fn increment_next_clone_index(&mut self, role_id: &AppRoleId) -> AppResult<()> {
+        self.role_mut(role_id)?.next_clone_index += 1;
+        Ok(())
+    }
```

Comment:
Looks like these functions are no longer used?

Reply:
They aren't, I forgot to delete them